packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -998,7 +998,6 @@ new PrismaClient({
         // it went bad, then we rollback the transaction
         await this._engine.transaction('rollback', headers, info).catch(() => {})
 
-        e.clientVersion = this._clientVersion
```

Comment:
Should we do something like

```ts
if (typeof e === 'object' && e !== null) {
  e.clientVersion = this._clientVersion
}
```

or was it never intended to be set here in the first place?

Reply:
(Looking at the new tests — apparently never intended to be set here; if the error was meant to have this field, it would have had it already set before re-throwing, if my understanding is correct. Marking the comment as resolved.)